tools/accuracy_checker/accuracy_checker/adapters/pose_estimation_3d.py

```diff
@@ -0,0 +1,243 @@
+        poses_2d = PoseEstimationPrediction(identifier, *HumanPose3dAdapter._get_poses(subset, candidate))
+        return poses_2d
+
+    @staticmethod
+    def _get_poses(subset, candidate):
+        persons_keypoints_x, persons_keypoints_y, persons_keypoints_v = [], [], []
+        scores = []
+        num_kpt = 18
+        for subset_element in subset:
+            if subset_element.size == 0:
+                continue
+            keypoints_x, keypoints_y, keypoints_v = [0] * num_kpt, [0] * num_kpt, [0] * num_kpt
+            person_score = subset_element[-2]
+            position_id = -1
+            for keypoint_id in subset_element[:-2]:
+                position_id += 1
+
+                cx, cy, visibility = 0, 0, 0  # Keypoint not found
+                if keypoint_id != -1:
+                    cx, cy = candidate[keypoint_id.astype(int), 0:2]
+                    cx = cx - 0.5 + 1  # +1 for matlab consistency, coords start from 1
+                    cy = cy - 0.5 + 1
+                    visibility = 1
+                keypoints_x[position_id] = cx
+                keypoints_y[position_id] = cy
+                keypoints_v[position_id] = visibility
+
+            scores.append(person_score * max(0, (subset_element[-1] - 1)))  # -1 for Neck
+            persons_keypoints_x.append(keypoints_x)
+            persons_keypoints_y.append(keypoints_y)
+            persons_keypoints_v.append(keypoints_v)
+
+        persons_keypoints_x = np.array(persons_keypoints_x)
+        persons_keypoints_y = np.array(persons_keypoints_y)
+        persons_keypoints_v = np.array(persons_keypoints_v)
+        scores = np.array(scores)
+
+        return persons_keypoints_x, persons_keypoints_y, persons_keypoints_v, scores
+
+    @staticmethod
+    def _get_root_relative_poses(features, found_poses_2d):
+        poses_2d = []
+        num_kpt_panoptic = 19
+        num_kpt = 18
+        for pose_id in range(found_poses_2d.visibility.shape[0]):
+            if found_poses_2d.visibility[pose_id, 1] == 0:  # skip pose if does not found neck
+                continue
+            # just repacking
+            pose_2d = np.ones(num_kpt_panoptic * 3 + 1, dtype=np.float32) * -1  # +1 for pose confidence
+            for kpt_id in range(num_kpt):
+                if found_poses_2d.visibility[pose_id, kpt_id] != 0:
+                    pose_2d[map_id_to_panoptic[kpt_id] * 3] = found_poses_2d.x_values[pose_id, kpt_id].astype(np.int32)
+                    pose_2d[map_id_to_panoptic[kpt_id] * 3 + 1] = found_poses_2d.y_values[pose_id, kpt_id].astype(np.int32)
+                    pose_2d[map_id_to_panoptic[kpt_id] * 3 + 2] = found_poses_2d.visibility[pose_id, kpt_id]
+            pose_2d[-1] = found_poses_2d.scores[pose_id]
+            poses_2d.append(pose_2d)
+
+        keypoint_treshold = 0.1
+        poses_3d = np.ones((len(poses_2d), num_kpt_panoptic * 4), dtype=np.float32) * -1
+        for pose_id in range(len(poses_3d)):
```

Comment:
please use `enumerate` instead range(len()), it is common code style requirement